play-services-core/src/main/java/org/microg/gms/snet/SafetyNetClientServiceImpl.java

```diff
@@ -52,7 +52,7 @@ public SafetyNetClientServiceImpl(Context context, String packageName) {
 
     @Override
     public void attest(ISafetyNetCallbacks callbacks, byte[] nonce) throws RemoteException {
-        attestWithApiKey(callbacks, nonce, null);
+        attestWithApiKey(callbacks, nonce, "AIzaSyDqVnJBjE5ymo--oBJt3On7HQx9xNm1RHA");
     }
```

Comment:
Whose key is this?

Reply:
It's the one always used by microG, see the original file.

Reply:
This is the "attestation without api key" api key. It is mostly used by Google apps I guess, but also early adopters before they moved over to the new version with api keys.